src/Components/PresentationalComponents/CSAwRuleBox/__snapshots__/CSAwRuleBox.test.js.snap

```diff
@@ -1,355 +1,13 @@
-                  gutter="md"
-                >
-                  <SplitItem>
-                    <Label
-                      className="label"
-                    >
-                      <FormattedMessage
-                        defaultMessage="Risk of change"
-                        description="Risk of change label for CVE rules"
-                        id="riskOfChange"
-                        values={Object {}}
-                      />
-                    </Label>
-                  </SplitItem>
-                  <SplitItem
-                    className="icon-with-label"
-                    isFilled={true}
-                  >
-                    <ExclamationTriangleIcon
-                      color="var(--pf-global--warning-color--100)"
-                      noVerticalAlign={false}
-                      size="sm"
-                      title={null}
-                    />
-                    Moderate
-                    <Tooltip
-                      appendTo={[Function]}
-                      aria="describedby"
-                      boundary="window"
-                      className=""
-                      content="These will likely require an outage window."
-                      distance={15}
-                      enableFlip={true}
-                      entryDelay={500}
-                      exitDelay={500}
-                      flipBehavior={
-                        Array [
-                          "top",
-                          "right",
-                          "bottom",
-                          "left",
-                          "top",
-                          "right",
-                          "bottom",
-                        ]
-                      }
-                      id=""
-                      isAppLauncher={false}
-                      isContentLeftAligned={false}
-                      isVisible={false}
-                      maxWidth="18.75rem"
-                      position="top"
-                      tippyProps={Object {}}
-                      trigger="mouseenter focus"
-                      zIndex={9999}
-                    >
-                      <OutlinedQuestionCircleIcon
-                        className="l-sm-spacer"
-                        color="var(--pf-global--secondary-color--100)"
-                        noVerticalAlign={false}
-                        size="sm"
-                        title={null}
-                      />
-                    </Tooltip>
-                    <br />
-                  </SplitItem>
-                </Split>
-              </TextContent>
-            </StackItem>
-          </Stack>
-        </GridItem>
-      </Grid>
-    </CardBody>
-  </Card>
-</Fragment>
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
 `;
 
-exports[`CSAW Rule Box Should render without rules  1`] = `<Fragment />`;
+exports[`CSAW Rule Box Should render without rules  1`] = `
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
+`;
```

Comment:
Is this file having empty components intentional?

Reply:
the snapshot is generated by the library.